packages/@react-aria/tabs/docs/useTab.mdx

```diff
@@ -0,0 +1,195 @@
+<!-- Copyright 2020 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License. -->
+
+import {Layout} from '@react-spectrum/docs';
+export default Layout;
+
+import docs from 'docs:@react-aria/tabs';
+import {HeaderInfo, FunctionAPI, TypeContext, InterfaceType} from '@react-spectrum/docs';
+import packageData from '@react-aria/tabs/package.json';
+import Anatomy from './anatomy.svg';
+
+```jsx import
+import {useTab, useTabList, useTabPanel} from '@react-aria/tabs';
+import {useTabListState} from '@react-stately/tabs';
+```
+
+---
+category: Navigation
+keywords: [tabs, aria]
+---
+
+# useTabList
```

Comment:
yeah, i have no idea which to use, because useTab is a hook for a specific tab, useTabs doesn't exist, tablist and tabpanel are the others
so what should the overall name be?